kafka-streams-partitioners/weighted-group-partitioner/src/main/java/org/hypertrace/core/kafkastreams/framework/partitioner/KeyHashPartitioner.java

```diff
@@ -0,0 +1,15 @@
+package org.hypertrace.core.kafkastreams.framework.partitioner;
+
+import org.apache.kafka.common.utils.Utils;
+import org.apache.kafka.streams.processor.StreamPartitioner;
+
```

Comment:
```suggestion
/**
 This a delegate stream partitioner and it should be used with conjunctions with WeightedGroupStreamPartioner. See more details here - https://github.com/hypertrace/kafka-streams-framework/blob/main/kafka-streams-partitioners/weighted-group-partitioner/src/main/java/org/hypertrace/core/kafkastreams/framework/partitioner/WeightedGroupPartitioner.java#L14
*/
```

Reply:
This by design is an independent partitioner and can be used standalone as well.
And we have multiple such variants: roundrobin, keyhash, valuehasg, fieldvalue etc.
So, I don't think this documentation is relevant here.

I will go ahead and add everywhere if you think otherwise.